Src/BootCamp.Chapter/MsFactory.cs

```diff
@@ -1,26 +1,27 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using BootCamp.Chapter.Computer;
+﻿using BootCamp.Chapter.Computer;
 
 namespace BootCamp.Chapter
 {
     public class MsFactory : DesktopComputer
```

Comment:
What dos Ms factory have different compared to Win factory? NOthing. Please make sure that Ms factory creates MsRam, MsBody, MsGpu, etc. And Mac creates MacRam, MacCpu, MacBody, etc.

Reply:
I removed the inheritance of DesktopComputer because these are not related I think?